rebar.config

```diff
@@ -50,21 +50,32 @@
  [
   {base16, "2.0.0"},
   {exml, "3.0.5", {pkg, hexml}},
+  {jiffy, "1.0.8"},
+  {jid, "1.0.0", {pkg, mongoose_jid}},
+  {tomerl, "0.5.0"},
+  {bbmustache, "1.12.2"},
+
   {telemetry, "1.0.0"},
-  {segmented_cache, "0.1.1"},
   {lager, "3.9.2"}, % We're keeping this to override the dependencies
-  {cowboy, "2.9.0"},
   {exometer_core, {git, "https://github.com/esl/exometer_core.git", {branch, "master"}}},
   {exometer_report_graphite, {git, "https://github.com/esl/exometer_report_graphite.git", {branch, "master"}}},
   {exometer_report_statsd, {git, "https://github.com/esl/exometer_report_statsd.git", {branch, "master"}}},
-  {fast_scram, "0.2.0"},
-  {jiffy, "1.0.8"},
-  {fusco, {git, "https://github.com/esl/fusco.git", {branch, "master"}}},
-  {idna, "6.0.1"},
-  {uuid, "1.8.0", {pkg, uuid_erl}},
-  {fast_tls, {git, "https://github.com/processone/fast_tls.git", {ref, "ceb277f"}}},
+  {syslogger, "*", {git, "https://github.com/NelsonVides/syslogger.git", {branch, "build_port_compiler"}}},
+  {flatlog, {git, "https://github.com/ferd/flatlog.git", {branch, "master"}}},
+
+  {cowboy, "2.9.0"},
+  {gun, "1.3.3"},
+  {fusco, "0.1.1"},
   {lasse, "1.2.0"},
-  {worker_pool, "4.0.3"},
+
+  {fast_scram, "0.4.3"},
+  {fast_tls, "1.1.13"},
+  {cache_tab, "1.0.29"},
+
+  {segmented_cache, "0.1.1"},
+  {idna, "6.1.1"},
```

Comment:
This could be maybe moved to the top?

Reply:
idna maybe, it's a pure library with no state and only one irrelevant dependency, but segmented_cache has a state (persistent term and a bunch of ets tables), I'd rather keep it like, next to cache_tab, another caching library 🤔

Reply:
I was thinking about idna, they even removed the dependency in the repository, but didn't publish a new package without it.

Reply:
Oh, so we gotta ping benoit and ask 🤔